project-board-crew-logic/src/main/java/project/board/crew/logic/structure/crew/Crew.java

```diff
@@ -0,0 +1,40 @@
+package project.board.crew.logic.structure.crew;
+
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import lombok.Setter;
+
+import project.board.crew.logic.structure.categories.CrewCategories;
+import project.board.crew.logic.structure.user.Users;
+
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.List;
+
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+public class Crew {
+
+    @Id
+    private Long id;
+
+    private String name;
+
+    @ManyToMany
+    List<Users> users;
+
+    @ManyToMany
+    List<CrewCategories> crewCategories;
+
+    public Crew() {}
```

Comment:
Jak już używasz lomboka, to nie możesz użyć @NoArgsConstructor czy coś w tym stylu? Chyba że JPA nie czyta takiego kontruktora to spoko